react-graph/src/index.ts

```diff
@@ -4,5 +4,6 @@ import LineWithThreshold from './LineWithThreshold';
 import HorizontalMarker from './HorizontalMarker';
 import Button from './Button';
 import VerticalMarker from './VerticalMarker';
+import Step from './Step';
 
-export {Plot, Line, LineWithThreshold, Button, HorizontalMarker, VerticalMarker};
+export { Plot, Line, LineWithThreshold, Button, HorizontalMarker, VerticalMarker, Step };
```

Comment:
I would rename this component `ProgressBar`. Also I wopuld move it into `react-interactive`